packages/docs/docusaurus.config.js

```diff
@@ -64,6 +64,12 @@ const config = {
             position: 'left',
             label: 'SDK',
           },
+          {
+            href: '/api',
+            target: '_blank',
```

Comment:
is this by design?

Reply:
Yes, added a link to the Redoc page (static html file).

See Slack thread [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1654334346129139?thread_ts=1654331790.882009&cid=C02A8G4HVAM)